packages/analytics/shippers/elastic_v3/common/src/build_headers.ts

```diff
@@ -0,0 +1,16 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+export function buildHeaders(clusterUuid: string, version: string, licenseId?: string) {
```

Comment:
NIT: I would go with:

```ts
{ clusterUuid, version, licenseId }: { clusterUuid: string; version: string; licenseId?: string; }
```

just in case we add additional options later on